canvas_modules/common-canvas/src/common-properties/properties-main/properties-main.jsx

```diff
@@ -420,11 +427,19 @@ class PropertiesMain extends React.Component {
 		this.setState({ showPropertiesButtons: state });
 	}
 
+	updateRightFlyoutWidth(size) {
+		const element = document.querySelector(".common-canvas .right-flyout-container");
```

Comment:
@nmgokhale Would appreciate your feedback on above I can see similar common properties dependancy on common canvas over [here](https://github.com/elyra-ai/canvas/blob/32f2a74d4d85844bb49e2f94b12b16c11aab81c8/canvas_modules/common-canvas/src/common-properties/properties-main/properties-main.scss#L73) 

Without enforcing width to `right-flyout` in `properties` it will look something like below, where the properties content width is set as per custom classnames but flyout width is not reset.

<img width="726" alt="Screenshot 2024-11-12 at 10 52 49 PM" src="https://github.com/user-attachments/assets/38e36c23-c280-4cae-a533-9717fc51ccf2">